src/domain/schemas/user-schemas.js

```diff
@@ -57,3 +57,37 @@ export const loginBodySchema = joi.object({
   email: joi.string().required(),
   password: joi.string().required()
 });
+
+const storageReminderDBSchema = joi.object({
+  timestamp: joi.date().required(),
+  createdBy: idOrKeySchema.required()
```

Comment:
I ask myself if this will ever end up being used, as we seem to not continue to develop any more storage allocation concepts..

Reply:
I don't understand, it is used :thinking: You mean the reminder, right?

Reply:
Yes, the reminder feature.

Reply:
It is used then. (in the sense that admins can set a marker for whom they've sent a reminder)

Reply:
I know they **can**, I just wandered if the **do** 😬

Reply:
I see. As they don't charge yet, I guess not. Questionable if ever :man_shrugging: